src/client/components/common/SecondaryButton.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import { Button, ActivityIndicator, Pressable, Text } from 'react-native';
+import { commonStyles, DEFAULT_COLOURS } from '../../styles/commonStyles'
+
+const SecondaryButton = ({label, onSubmit, buttonStyle, textStyle}) => {
```

Comment:
this seems to be exactly the same as the `<SubmitButton>` component with just additional props (`buttonStyle`, `textStyle`), why dont we make both of these components into one generic `Button` component that just accepts these two props as optional?

- allows us to change color based off prop value (`style={[customButtonStyle, buttonStyle]}`) where `customButtonStyle` is passed as a prop
- we can render `textStyle` instead of `commonStyles.submitText` wiith a simple undefined check (e.g. `textStyle ? textStyle : commonStyles.submitText`

```
interface ButtonProps {
  label: string;
  onSubmit: () => void;
  buttonStyle?: string;
  textStyle?: TextStyle;
}